code/modules/martial arts/wrestling.dm

```diff
@@ -199,7 +199,11 @@ You can also climb tables by dragging and dropping yourself on them!<br>
 	armor_block = A.run_armor_check(affecting, "melee")
 	A.apply_effect(4, WEAKEN)//, armor_block)
 	add_logs(A, D, "suplexed", addition="(Wrassling)")
-	playsound(D, pick("swing_hit"), 40, 1)
+	if(prob(50))
+		playsound(D, pick("swing_hit"), 40, 1)
```

Comment:
the only thing wrong with it is that it's a pick()
anyway playsound has pre-defined sound lists for a bunch of common shit like swing_hit, etc. that can be accessed via strings